src/styles/_catalog.scss

```
@@ -20,3 +21,28 @@
     margin-right: 48px;
   }
 }
+
+@media (min-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(4, 1fr);
```

Use `auto` because `1fr` takes up all the possible width

```suggestion
    grid-template-columns: repeat(4, auto);
```

---

Круто, не знала про auto!)) Дякую. Виправила.